openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/common.py

```diff
@@ -126,45 +123,45 @@ def build_source(ingestion_pipeline: IngestionPipeline) -> WorkflowSource:
             metadata.health_check()
             return WorkflowSource(
                 type=service_type,
-                serviceName=ingestion_pipeline.service.name,
+                serviceName=quote_name(ingestion_pipeline.service.name),
                 sourceConfig=ingestion_pipeline.sourceConfig,
                 serviceConnection=None,  # retrieved from the test suite workflow using the `sourceConfig.config.entityFullyQualifiedName`
             )
 
         if service_type == "databaseService":
             entity_class = DatabaseService
             service: DatabaseService = metadata.get_by_name(
-                entity=entity_class, fqn=ingestion_pipeline.service.name
+                entity=entity_class, fqn=quote_name(ingestion_pipeline.service.name)
```

Comment:
instead of modifying all the get_by_name, why not change it in the method itself?

Reply:
Yes we can do that